apps/pie-storybook/stories/pie-input.stories.ts

```diff
@@ -194,6 +194,13 @@ const inputStoryMeta: InputStoryMeta = {
             control: 'select',
             options: sizes,
         },
+        required: {
+            description: 'If true, the input is required to have a value before submitting the form. The form will autofocus a required field if it is missing a value. Will have no effect if the input is placed inside a form.',
```

Comment:
is this accurate? 

```
Will have no effect if the input is placed inside a form.
```

Reply:
In readme and defs too, as I'm sure you're aware

Reply:
lol! yeah thanks for spotting. will update now 👍 🚀